packages/runner/lib/exec.ts

```diff
@@ -87,6 +87,19 @@ export async function exec(
                     if (typeof codeParams === 'object' && Object.keys(codeParams).length === 0) {
                         inputParams = undefined;
                     }
+
+                    // Validate action input against json schema
+                    if (inputParams) {
+                        const val = validateInput({
+                            input: inputParams,
+                            modelName: nangoProps.syncConfig.input,
+                            jsonSchema: nangoProps.syncConfig.models_json_schema
+                        });
+                        if (val !== true) {
```

Comment:
`if(!val)` no?

Reply:
it's `true` when it's valid and an array of errors when not valid.
But `!val` is not the opposite of `not true`, I could put Array.isArray to be clearer (and I can't use ok/err because it's in utils and I need that function in the CLI)